docs/_static/monitoring/v1.yaml

```diff
@@ -0,0 +1,65 @@
+openapi: "3.0.0"
+info:
+  title: Nuts Node Monitoring API spec
+  description: API specification for monitoring of the nuts node
+  version: 1.0.0
+  license:
+    name: GPLv3
+servers:
+  - url: http://localhost:1323
+paths:
+  /status:
+    get:
+      summary: "healthcheck to confirm node is up and running"
+      description: |
+        Check if the node is up and running.
+      operationId: getStatus
+      tags:
+        - status
+      responses:
+        200:
+          description: '"OK" when the node is up and running'
+          content:
+            text/plain:
+              schema:
+                type: string
+  /status/diagnostics:
+    get:
+      summary: "get node diagnostics"
+      description: |
+        Get basic information on the
+        * connected peers
+        * DAG state
+        * node build info
+      operationId: getDiagnostics
+      tags:
+        - status
+      responses:
+        200:
+          description: Basic nuts-node diagnostics
+          content:
+            application/json:
```

Comment:
It returns it in YAML, unless `application/json` is specified in the `Accept` header

Reply:
The default `Accept` header only contains `text/plain`. Should this be changed to reflect that it is yaml as well?

Reply:
- Maybe: respond with `Content-Type: application/yaml` when returned in YAML (instead of text/plain)
- Return YAML content-type when requested with Accept